src/control/cmd/dmg/pretty/pool.go

```diff
@@ -85,3 +87,162 @@ func PrintPoolCreateResponse(pcr *control.PoolCreateResp, out io.Writer, opts ..
 
 	return err
 }
+
+func poolListCreateRow(pool *control.Pool) txtfmt.TableRow {
+	name := pool.Label
+	if name == "" {
+		// use short version of uuid if no label
+		name = strings.Split(pool.UUID, "-")[0]
```

Comment:
Shall we keep a trailing "-" at the end of short UUID to show that this is a UUID? AFAIK, "-" is forbidden in the label.
@mjmac any thoughts?

Reply:
in discussion with @mjmac and @johannlombardi decided not to print trailing "-" for the moment